ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -140,7 +140,8 @@ Delete Data Science Projects From CLI
             Log    msg=There is probably no DS Projects with Diplay Name equal to ${displayed_name}
             Continue For Loop
         ELSE
-            Oc Delete    kind=Project   name=${project_k8s_name}
+            # the project can be deleted in the test case
+            Run And Return Rc    oc delete project ${project_k8s_name} || true
```

Comment:
In the past we were prioritizing usage of the robot framework OpenShift library instead of using the keyword `Run`. Is this change fixing any issue?

Reply:
`Oc Delete` will fail if the project is not there

Reply:
the project is expected to be there, so we should know if there's something else deleting the project before the expected moment. I would revert the change

Reply:
Added `skip_on_failure`

Reply:
why do we need this change?

Reply:
got it. 
There is already this piece of code which skips the deletion command if the project doesn't exist:
```
${project_k8s_name}=     Run     oc get projects -o json | jq '.items[] | select((.metadata.annotations."openshift.io/display-name" != null) and (.metadata.labels."opendatahub.io/dashboard"=="true") and (.metadata.annotations."openshift.io/display-name"=="${displayed_name}")) | .metadata.name'    # robocop: disable
        ${project_k8s_name}=    Replace String    ${project_k8s_name}    "    ${EMPTY}
        IF    "${project_k8s_name}" == "${EMPTY}"
            Log    msg=There is probably no DS Projects with Diplay Name equal to ${displayed_name}
            Continue For Loop
```